src/VirtoCommerce.Platform.Web/Controllers/Api/PlatformExportImportController.cs

```diff
@@ -368,12 +390,14 @@ void progressCallback(ExportImportProgressInfo x)
                 {
                     Directory.CreateDirectory(localTmpFolder);
                 }
-                if (System.IO.File.Exists(localTmpPath))
+
+                if (SystemFile.Exists(localTmpPath))
```

Comment:
Seems such an alias for System.IO.File using reduce readability, it looks like some unknown new class called `SystemFile` instead of the well-known System.IO.File. So I'd better leave the previous version.

Reply:
We cannot use the `File` class insider of the controller, because File is a method from the controller. So we have two variants:
1. Use `System.IO.File`;
2. Make an alias for it;

I think the second way is more clear.

Reply:
Imho the `System.IO.File` is more clear. It is well known for almost every .NET developer, and `SystemFile` just raises the questions.
So I'd leave this as before.

Reply:
Reverted.